### DISCUSSION TITLE
Add Missing Ship & Remove Tidal Black Stone

### DISCUSSION THREAD

**Discussion by MohammadAdiyudha** [xanthics/bdo_ship_upgrade#7] [Ideas]
Hello

I always use your web to track my ship progression because the web is fast & simple, but i just notice you missing 1 ship from BDO, the name is Panokseon. It is a ship from BDO region called Land of Morning Light.

More information about materials & how to get them : 
- https://grumpygreen.cricket/panokseon/
- https://www.blackdesertfoundry.com/panokseon-ship-guide/

And also, if possible i think you should remove all required tidal black stone in all of ship materials, because black stone at the end of the day still black stone, it matters of RNG of the players, so we cant predict how many tidal black stone needed to enchant ship gear.

Thank you very much.

**Comment by xanthics** [comment]
> And also, if possible i think you should remove all required tidal black stone in all of ship materials, because black stone at the end of the day still black stone, it matters of RNG of the players, so we cant predict how many tidal black stone needed to enchant ship gear.

The number included is the minimum and contains notes to that effect.  Or are you saying they completely changed it?

I'll add the other ship tomorrow when I have some time.

**Reply by MohammadAdiyudha** [reply]
ah.. if its minimum, then ok keep it as it is,

**Comment by xanthics** [comment]
https://github.com/xanthics/bdo_ship_upgrade/commit/c5d263d41fbb0dcc2d284c56205eadf8f97abfdf

**Reply by MohammadAdiyudha** [reply]
thankyou very much bro
